order/src/main/java/com/caldevsupplychain/order/vo/OrderWS.java

```diff
@@ -0,0 +1,29 @@
+package com.caldevsupplychain.order.vo;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.List;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class OrderWS implements Serializable {
+
+	private static final long serialVersionUID = -8744630767752497964L;
+
+	private String uuid;
+	private String displayId;
+	private Long id;
```

Comment:
remove `private Long id;`. `uuid` is the id we expose to public.
use `uuid` as `userId` and `agentId`. maybe name them userUuid to make it clear

Reply:
Fixed